README.md

```diff
@@ -1,17 +1,31 @@
-# nvim-dap-reactnative
-Standalone debug adapter protocol for react native. Hopefully won't have any specific editor/ide stuff
+<h1 align="center">Welcome to NVIM Dap React Native 👋</h1>
+<p>
+  <img alt="Version" src="https://img.shields.io/badge/version-1.9.1-blue.svg?cacheSeconds=2592000" />
+  <a href="https://github.com/microsoft/vscode-react-native#readme" target="_blank">
+    <img alt="Documentation" src="https://img.shields.io/badge/documentation-yes-brightgreen.svg" />
+  </a>
+  <a href="https://github.com/microsoft/vscode-react-native/graphs/commit-activity" target="_blank">
+    <img alt="Maintenance" src="https://img.shields.io/badge/Maintained%3F-yes-green.svg" />
```

Comment:
As we are not actively maintaining yet, I am not comfortable having this maintained as `Yes` as of now. If we keep using this everyday and keep fixing issues people face for like few months, we can call this maintained. Else we don't know at all if something is broken.

Reply:
I have removed the badges as they are not correct.